views/update/remove-need-to-make-changes-to-entity.html

```diff
@@ -0,0 +1,76 @@
+{% extends "update-layout.html" %}
```

Comment:
Potentially, no new template required here - see earlier comments and take a look at `update-interrupt-card.html` where an existing Update screen content is varied depending on the journey.